wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_streamfield.scss

```diff
@@ -133,6 +156,7 @@ li.sequence-member {
 
 // This horridly specific selector ensures text inputs, rich text fields and textareas
 // that are direct children of highest level sequence should be borderless and full-width
+// Which is almost a terrible idea @TODO
```

Comment:
Those style overrides make it impossible to see there is a field. Could we just remove the whole block?